torch/csrc/autograd/python_variable.h

```diff
@@ -20,7 +20,7 @@ struct THPVariable {
 THP_API PyObject *THPVariableClass;
 
 bool THPVariable_initModule(PyObject *module);
-PyObject * THPVariable_Wrap(torch::autograd::Variable var);
+THP_CLASS PyObject * THPVariable_Wrap(torch::autograd::Variable var);
```

Comment:
Why are we using `THP_CLASS` to decorate functions? Can we make a new macro?

Reply:
Can you give an example? The C type export decoration is named `THP_API` already.

Reply:
Then let's use `THP_API`. `THP_CLASS` is confusing when not used with a class